src/material-experimental/mdc-input/input.spec.ts

```diff
@@ -1823,3 +1865,14 @@ class MatInputWithColor {
   `
 })
 class MatInputInsideOutsideFormField {}
+
+@Component({
+  template: `
+    <mat-form-field>
+      <input matInput required [formControl]="formControl">
```

Comment:
I think that the `ReactiveFormsModule` discourages using `required` (and similar attributes) together with `formControl`, in favor of passing in `Validator` functions. That being said, we currently have an issue where we can't detect `Validators.required` from a `FormControl` because `@angular/forms` doesn't have an API for it. Are these tests capturing something that isn't captured by the other `ngModel`-based tests?

Reply:
Assuming I'm reading the tests correctly, what exists right now does a good job of testing a `required` `matInput` and a good job of testing a `matInput` which relies on the `ReactiveFormsModule` without the native `required` attribute.

These specifically test when both are present since the new logic considers both when assigning `aria-required`.  If we are okay with leaving that scenario untested since it's discouraged by `ReactiveFormsModule` I'm fine to remove these tests.  I included them for completeness' sake more than anything.

Reply:
These `ReactiveFormsModule` tests can safely be removed since they don't reflect how the component should be used.

Reply:
These tests have been removed.